tests/karma/unit/directives/enter.js

```diff
@@ -0,0 +1,44 @@
+describe('enter directive', function() {
+
+  'use strict';
+
+  var compile,
```

Comment:
Don't use `var` in ES6 capable files - in this case use `let`. Also feel free to use the ES6 function syntax throughout.